design/object-bucket/object-bucket.md

```diff
@@ -63,7 +53,7 @@ _As an admin, I want to expose an existing Ceph object store to users so they ca
 
 ![admin-actions](./cobc-admin.png)
 
-1. The admin creates a [StorageClass](#storageclass) with the `provisioner=ceph.rook.io/object`, `parameters[serviceName]=OBJECT-STORE-SERVICE`, and `parameters[serviceNamespace]=OBJECT-STORE-NAMESPACE` key-values.
+1. The admin creates a [StorageClass](#storageclass) with the `provisioner=ceph.rook.io/object`, `parameters[objectStore]=OBJECT-STORE`, and `parameters[objectStoreNamespace]=OBJECT-STORE-NAMESPACE` key-values.
```

Comment:
Is OBJECT-STORE the name of the object store CR or it's service?

Reply:
the object store.  this was a change we made a while ago.  the service name is the same as the store.  From UX PoV, it makes more sense to me to have admins provide the object store.